test-frame-common/src/main/java/io/skodjob/testframe/clients/KubeClient.java

```diff
@@ -5,24 +5,27 @@
 package io.skodjob.testframe.clients;
 
 import io.fabric8.kubernetes.client.Config;
+import io.fabric8.kubernetes.client.ConfigBuilder;
 import io.fabric8.kubernetes.client.KubernetesClient;
 import io.fabric8.kubernetes.client.KubernetesClientBuilder;
 import io.fabric8.openshift.client.OpenShiftClient;
+import io.skodjob.testframe.TestFrameEnv;
+import io.skodjob.testframe.executor.Exec;
+
+import java.util.Arrays;
 
 public class KubeClient {
 
     private KubernetesClient client;
-    private static KubeClient kubeClient;
+    private String kubeconfigPath;
 
-    private KubeClient() {
-        this.client = new KubernetesClientBuilder().withConfig(Config.autoConfigure(System.getenv().getOrDefault("TEST_CLUSTER_CONTEXT", null))).build();
-    }
+    public KubeClient() {
+        Config config = getConfig();
 
-    public static KubeClient getInstance() {
-        if (kubeClient == null) {
-            kubeClient = new KubeClient();
-        }
-        return kubeClient;
+        this.client = new KubernetesClientBuilder()
+                .withConfig(config)
+                .build()
+                .adapt(OpenShiftClient.class);
```

Comment:
Does it need to be `.adapt(OpenShiftClient.class)`? Because we have here the whole method for that
```
public OpenShiftClient getOpenShiftClient() {
        return client.adapt(OpenShiftClient.class);
    }
```